helpers/clang.py

```diff
@@ -804,6 +804,8 @@ def help(lines):
     if matches:
         if matches.group[0] == "main":
             response = ["Did you try to compile a file that doesn't contain a `main` function?"]
+            if len(lines) > 3 and "helpers" in lines[2]:
```

Comment:
Let's be more precise with the regexes then, since just checking for simple substrings feels unnecessarily lax of us (in general). E.g., I'd match on `make: *** [helpers] Error`, unless that might vary.

And let's also include comments atop blocks like this to remind us what kind of output we're looking for?